documentation/modules/exploit/unix/ftp/proftpd_modcopy_exec.md

```diff
@@ -0,0 +1,94 @@
+## Vulnerable Application
+
+This module exploits the SITE CPFR/CPTO mod_copy commands in ProFTPD version 1.3.5.
+Any unauthenticated client can leverage these commands to copy files from any
+part of the filesystem to a chosen destination. The copy commands are executed with
+the rights of the ProFTPD service, which by default runs under the privileges of the
+'nobody' user. By using /proc/self/cmdline to copy a PHP payload to the website
+directory, PHP remote code execution is made possible.
+
+
+## Installation Steps
+
+Download and build:
```

Comment:
I noticed these do not come installed by default in Ubuntu 22. Did you use the following when installing 1.3.5 / would it be worth adding these to the Installation Steps? 
```
sudo apt install make 
sudo apt-get install gcc
```

Reply:
Added.

Depending on the distro there may be a bunch of other things which need to be installed (libc?), and using `apt install` won't work for systems which don't use Apt for package management or old systems which require using `apt-get install` instead.

The installation steps outline the basic steps to install, in particular any non-default and non-obvious steps users might encounter. Documenting the `--with-modules=mod_copy` configuration flag is extremely useful here, as is commentary surrounding creation of a world-writable directory within the web root.

During installation, the former would be identified by reading the vendor installation documentation. The latter would be identified by reading the module source code to understand the attack vector. Neither are intuitive.